seedfarmer/mgmt/module_init.py

```diff
@@ -19,18 +19,73 @@
 import yaml
 from cookiecutter.main import cookiecutter
 
-import seedfarmer.errors
 from seedfarmer import config
 
 _logger: logging.Logger = logging.getLogger(__name__)
 
 
+def remove_leading_path_sep(path: str) -> str:
+    if path.startswith(os.sep):
+        return str(path[len(os.sep) :])
+    # On Windows, also check for altsep ('/')
+    if os.altsep and path.startswith(os.altsep):
+        return str(path[len(os.altsep) :])
+    return path
+
+
+def add_module_manifest(module_name: str, module_path: str) -> None:
```

Comment:
question: can we achieve the same by adding the group file to the cookiecutter template?